assets/style.css

```diff
@@ -19,6 +19,48 @@ body {
   font-family: roboto, arial, sans-serif;
 }
 
+#toTopBtn {
+  position: fixed;
+ bottom: 26px;
+ right: 39px;
+ z-index: 98;
+ padding: 21px;
```

Comment:
Remove this padding and replace it with `width: 42px; height: 42px` which will maintain the same size but with actual dimensions and not just padding